projects/plugins/boost/app/assets/src/js/pages/critical-css-advanced/critical-css-advanced.tsx

```diff
@@ -61,7 +61,7 @@ export default function AdvancedCriticalCss() {
 	};
 
 	return (
-		<div className="jb-container--narrow jb-critical-css__advanced">
```

Comment:
This class is used in the E2E test at `tests/e2e/lib/pages/wp-admin/JetpackBoostPage.js`. Can you please review whether this needs to be kept here for the test, or if the test needs updating?

Reply:
Nice catch! I keep forgetting about e2e tests...

I've added the class back in so tests should be happy again.